hack/vendor/olm-test-script/e2e-olm.sh

```diff
@@ -83,26 +95,23 @@ $CSV
   packages: |-
 $PKG
 EOF
-CSV_CHANNEL=$(sed -nr 's,.*name: \"?([^"][^"]*)\"?,\1,p' $MANIFEST_DIR/*package.yaml)
-CURRENT_CSV=$(sed -nr 's,.*currentCSV: (.*),\1,p' $MANIFEST_DIR/*package.yaml)
-PACKAGE_NAME=$(sed -nr 's,.*packageName: (.*),\1,p' $MANIFEST_DIR/*package.yaml)
+
+CSV_CHANNEL=$($GNUSED -nr 's,.*name: \"?([^"][^"]*)\"?,\1,p' $MANIFEST_DIR/*package.yaml)
+CURRENT_CSV=$($GNUSED -nr 's,.*currentCSV: (.*),\1,p' $MANIFEST_DIR/*package.yaml)
+PACKAGE_NAME=$($GNUSED -nr 's,.*packageName: (.*),\1,p' $MANIFEST_DIR/*package.yaml)
 
 oc create -n $TEST_NAMESPACE -f /tmp/configmap.yaml
 if [ "${CREATE_OPERATORGROUP}" == "true" ] ; then
-  if [ "${GLOBAL}" == "true" ] ; then
-    oc -n ${TARGET_NAMESPACE} process -f "$(dirname $0)/operatorgroup-template-global.yaml" -p SUFFIX="${SUFFIX:-}" | oc create -n $TEST_NAMESPACE -f -
+  if [ "${TARGET_NAMESPACE}" = all ] ; then
+    oc process -f "$(dirname $0)/operatorgroup-allnamespaces-template.yaml" | oc create -n $TEST_NAMESPACE -f -
```

Comment:
this file was changed but I dont see it in the commit

Reply:
Removed the script updates, as the shawn-hurley/olm-test-script#5 is not urgent and not merged yet.